lib/ts/recipe/session/types.ts

```diff
@@ -49,6 +49,7 @@ export type CreateOrRefreshAPIResponse = {
     session: {
         handle: string;
```

Comment:
Need to add recipeUserId to:
- SessionInformation type
- createNewSession function input (optional recipeUserId. If not present, then recipeUserId = userId)
- getGlobalClaimValidators function
- regenerateAccessToken return type